src-tests/test-rpc.lisp

```diff
@@ -7,87 +7,110 @@
 
 (defparameter *expected-response* "test-response")
 
-(defun test-method (&key (sleep 1 sleep-p))
-  (when sleep-p
+(defun test-method (&key (sleep 1 slee-p))
```

Comment:
Couldn't resist a good pun, eh? A man after my own heart.  A tad gratuitous, but I'll allow it.